### ISSUE TITLE
Radicale 3 support

### ISSUE THREAD

**Issue by nebulade** [marcoh00/radicale-auth-ldap#11]
Hi,

we at Cloudron are in the process of updating our app package to Radicale v3 and thus were hoping to use this plugin for user integration, however it appears the Radicale auth plugin interface has slightly changed. I have created a custom plugin within our app package repo at https://git.cloudron.io/cloudron/radicale-app/-/tree/master/radicale_cloudron_ldap_auth

It would be great if we could somehow merge ours into this plugin to join forces.
Is there any interest within this project to support Radicale 3 and thus would a pull request with the fixes have any chances?

Thanks!

**Comment by marcoh00** [comment]
Hi!

I'm always happy to merge pull requests which improve the project. Please note that this plugin is mostly in maintenance mode, which means I haven't touched the code in ages! I'm okay with maintaining the repository though and help new contributors where I can.

In case you are actively using and maintaining this I could also imagine transferring the ownership of this repo.

**Comment by nebulade** [comment]
Thanks for your answer. I was taking a quick look how I could create a pull-request for radicale 3 support, however it seems their plugin system has diverged too much unfortunately. Just taking a look at the entrypoint in setup.py reveals major differences https://git.cloudron.io/cloudron/radicale-app/-/blob/master/radicale_cloudron_ldap_auth/setup.py#L3 vs https://github.com/marcoh00/radicale-auth-ldap/blob/master/setup.py#L3

Do you have an idea how this could be done to still keep your repository as the main repo to avoid confusion for users?
For example master branch could start supporting radicale 3 and a radicale 2 branch is created?

**Comment by marcoh00** [comment]
I created a `2-final` tag for users still interested in the Radicale 2 plugin. Feel free to submit your pull request.

Are you the original author of the v3 plugin or is it based on someone else's work? Anyhow, it would be nice if you could clarify authorship and the license of your code inside your `setup.py` and inside a comment at the top of the source file. Even though this repository had no `LICENSE` file until yet, it would be welcome as well :)

**Comment by nebulade** [comment]
Yes the code is written by me, but the original version was inspired by your code, thus the mostly matching config variable names. Which license would you prefer to be there? 

Sorry that progress on this is slow, hopefully I get around doing this this week.

**Comment by sphakka** [comment]
Hi!
What's the status? I'd really like to test it with Radicale 3.1.8 (Debian) :-)
